gaia-sdk-python/gaia_sdk/api/IdentityOp.py

```diff
@@ -29,5 +50,24 @@ def export(self, identity_id: str) -> Observable[bytes]:
             .pipe(
             ops.map(lambda response: response.content))
 
-    def import_identity(self, identity_id: str = None) -> None:  # 'import' keyword already taken
-        raise NotImplementedError("Implement identity import functionality is not yet implemented")
+    def import_identity(self, tenant_id: str, identity_name: str, content: bytes, override: bool = False,
+                        identity_id: str = None) -> Observable['IdentityOp']:
+        uri = "gaia://{}/identities/".format(tenant_id)
+        file_uri = DataRef.concat_uri(uri, identity_name)
+        number_of_chunks = ceil(len(content) / CHUNK_SIZE)
+
+        self._logger.debug(f"Started upload to uri {uri}")
+        new_file_data_ref = DataUpload(file_uri, content, number_of_chunks, override) \
+            .execute(self._client, self._scheduler)
+
+        self._logger.debug(f"Started import of identity {identity_name}")
+        self.complete_import(file_uri, tenant_id, identity_name, True, identity_id)
```

Comment:
mit dem observable von DataRef.add arbeiten, so dass der Import asynchron implementiert ist.
also zB
`return dataRef.add(...).pipe(
            ops.flat_map(lambda response: self.complete_import(....)),
            ops.do_on_next(lambda next: self._logger.debug("Finished import of identity with id ... to tenant ...")))`

Das do_on_next is eher Pseudocode - weiß nicht genau wie das in python aussieht